msal/src/test/java/com/microsoft/identity/client/robolectric/tests/AcquireTokenBaseTest.java

```diff
@@ -0,0 +1,54 @@
+package com.microsoft.identity.client.robolectric.tests;
+
+import android.app.Activity;
+import android.content.Context;
+
+import androidx.test.core.app.ApplicationProvider;
+
+import com.microsoft.identity.client.IPublicClientApplication;
+import com.microsoft.identity.client.PublicClientApplication;
+import com.microsoft.identity.client.exception.MsalException;
+
+import org.mockito.Mockito;
+
+import java.io.File;
+
+public abstract class AcquireTokenBaseTest {
+
+    private static final String CONFIG_FILE_PATH = "src/test/res/raw/aad_test_config.json";
```

Comment:
Are you planning to cover test functions specific to SingleAccountPCA or MultipleAccountPCA? I assume those function will be used more by the majority of apps. (AcquireToken with parameter is sort of 'advanced') 

Doesn't have to be in this scope - just FYI - we can chat later.

Reply:
Yeah, we can add those in later